Makefile

```diff
@@ -1,15 +1,22 @@
-all: build
+all: test
 
 ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 TAG = v0.2.2
+GOLANGCI_RELEASE = v1.4.1
 
-.PHONY: all build container clean
+.PHONY: all build container clean gofmt lint test
 
-build: test clean gofmt
+bootstrap:
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s $(GOLANGCI_RELEASE)
```

Comment:
go get/install is preferred over curl and bash
`go get -u github.com/golang/lint/golint` ?

Especially since it's not recommended for CI https://github.com/golangci/golangci-lint#local-installation :(